index.html

```diff
@@ -2090,15 +2121,15 @@ <h3 id="applying">
           <code>orientation</code> member
         </h3>
         <p>
-          The <dfn id="member-orientation"><code>orientation</code>
-          member</dfn> is a <a>string</a> that serves as the <a>default
+          The <dfn>orientation</dfn>
```

Comment:
Nit: the `<dfn>orientation</dfn>` and so on, should all be defined inside the `h3`s... like this:

```HTML
<h3>
  <dfn>orientation</dfn> member 
</h3>
<p>The <a>orientation</a> member bla bla..</p>
```

Applies to all of the ones below...

Reply:
but that makes them all cursive :(